aims/aim1.md

```diff
@@ -0,0 +1,44 @@
+# Aim 1:
+
+**Develop proof-of-concept deep learning methods to learn a latent space from single cell transcriptomic data.**
+
+## Background
+
+Deep unsupervised models can learn to approximate a data generating manifold often represented in a lower dimensional latent space.
```

Comment:
Do you think you might port over some text from Aim 3 that might fit in well here? Specifically I'm thinking about the opening of your background in [Aim 3](https://github.com/greenelab/czi-rfa/blob/master/aims/aim3.md)

> Latent features describe a lower dimensional representation of input data and there are many methods, including principal components analysis (PCA) and non-negative matrix factorization (NMF), that arrive at unique solutions. Certain classes of deep generative models, including variational autoencoders (VAEs) and generative adversarial networks (GANs), that add a distribution matching constraint, imbue intuitive mathematical features to the learned latent features [1,2].